lib/utils.js

```diff
@@ -56,9 +43,9 @@ utils.isPath = function isPath(pattern, opts) {
  * @return {Function}
  */
 
-utils.hasPath = function hasPath(pattern, opts) {
+utils.pathContains = function(pattern) {
   return function(fp) {
-    return utils.unixify(pattern, opts).indexOf(fp) !== -1;
```

Comment:
> Is unixify being dropped in these two utils because it's happening in matcher() now instead?

right, I stepped through the code as carefully as I could to make sure paths were being unixified everywhere. we were doing it twice there.